docs/server-side-rendering.md

```diff
@@ -20,6 +20,29 @@ React components used on the server will be rendered to HTML by being passed to
 * Libraries that are used on the server should be mindful of the DOM not being available on the server, and should either: be modified to work without the DOM; have non-DOM specific fallbacks; or fail in an obvious manner.
 * Singletons should be avoided, as once instantiated they will persist for the entire duration of the `node` process.
 
+### Caching
+
+Because it is necessary to serve the redux state along with a server-rendered page, we use two levels of cache on the server—one to store raw query data, from which we can generate and serve redux state, and one to store rendered layouts.
```

Comment:
I read `server-one` as an adjective at first -- can we use a colon and space instead of what looks like a hyphen here?